lib/matplotlib/tests/test_mathtext.py

```diff
@@ -385,3 +385,14 @@ def test_math_fontfamily():
              size=24, math_fontfamily='dejavusans')
     fig.text(0.2, 0.3, r"$This\ text\ should\ have\ another$",
              size=24, math_fontfamily='stix')
+
+
+def test_mathtext_cmr10_minus_sign():
+    # cmr10 does not contain a minus sign and used to issue a warning
+    # RuntimeWarning: Glyph 8722 missing from current font.
+    mpl.rcParams['font.family'] = 'cmr10'
```

Comment:
> Fortunately, when using mathtext, you can use multiple fonts at the same time; so one needs to force the ticks to use mathtext with rcParams["axes.formatter.use_mathtext"] = True, and then...

The tests will succeed if you force ticks to use `mathtext`, following https://github.com/matplotlib/matplotlib/issues/17007#issuecomment-685951519:

```suggestion
    mpl.rcParams['font.family'] = 'cmr10'
    mpl.rcParams["axes.formatter.use_mathtext"] = True
```

Reply:
nice... I can confirm that additional config fixes the issue. Basically users with `font.family == 'cmr10'` also need `axes.formatter.use_mathtext == True`.

(This PR is still required to ensure this works).
Using slightly different suggestion for consistency with quotes:

```suggestion
    mpl.rcParams['font.family'] = 'cmr10'
    mpl.rcParams['axes.formatter.use_mathtext'] = True
```